docs/integrations/notification/_index.rst

```diff
@@ -140,19 +165,33 @@ To create a webhook:
    If you do not have sufficient permissions to view and create webhooks, consult with a systems
```

Comment:
To create a webhook, follow these steps:

... note::
 If you do not have sufficient permissions to view and create webhooks, consult with a systems administrator.